prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -978,6 +979,9 @@ module dftbp_initprogram
   !> Is this DFTB/SSR formalism
   logical :: tREKS
 
+  !> data type for REKS
+  type(TReksCalc), save :: reks
```

Comment:
Can you make this allocatable (as is done for other variables like sccCalc)? The allocated status then shows the type of calculation (and also in later Fortran standards allows it to be combined with optional arguments for subroutines). Also, please remove the save attribute, as this will need to be removed to make the code thread safe for the API.